apps/api/src/resources/swagger.yaml

```diff
@@ -34,3 +34,152 @@ paths:
             application/json:
               schema:
                 type: object
+
+  /application/edit:
+    post:
+      tags:
+        - Applications
+      summary: Edit application details
+      requestBody:
+        description: Changes made to an Application record
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApplicationUpdates'
+      responses:
+        '500':
+          description: 'Error editing application'
+          content:
+            application/json:
+              schema:
+                type: object
+        '400':
+          description: 'Error editing application - Application not found'
+          content:
+            application/json:
+              schema:
+                type: object
+        '200':
+          description: 'Application updated'
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
responses schemas can also be defined here using the structures defined by the `success` and `failure` helpers

Reply:
Will add in #58